<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>300</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Crear usuario</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../img/icon.png</normaloff>../img/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#555860; color: black; border-radius: 10px</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="user">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #e9e8eb; color: black; border-radius: 5px;</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>9</number>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="placeholderText">
     <string notr="true">DNI</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwd">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #e9e8eb; color: black;border-radius: 5px;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="placeholderText">
     <string notr="true">Contraseña</string>
    </property>
   </widget>
   <widget class="QPushButton" name="newUserButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>201</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #222628; color: white;</string>
    </property>
    <property name="text">
     <string>Crear usuario</string>
    </property>
   </widget>
   <widget class="QLabel" name="titulo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white; font-weight: bold; font-size: 25px;</string>
    </property>
    <property name="text">
     <string>NUEVO USUARIO</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="adminCheck">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>240</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Administrador</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="repeat_pass">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #e9e8eb; color: black;border-radius: 5px;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="placeholderText">
     <string notr="true">Repetir contraseña</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>user</tabstop>
  <tabstop>passwd</tabstop>
  <tabstop>repeat_pass</tabstop>
  <tabstop>newUserButton</tabstop>
  <tabstop>adminCheck</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
